pkgs/build-support/rust/default.nix

```diff
@@ -29,7 +30,7 @@ assert buildType == "release" || buildType == "debug";
 let
   cargoDeps = if cargoVendorDir == null
     then fetchcargo {
-        inherit name src srcs sourceRoot cargoUpdateHook;
+        inherit name src srcs sourceRoot unpackPhase cargoUpdateHook;
```

Comment:
It's just a bit counter-intuitive that `unpackPhase` is executed on the `-vendor` directory and not on the package itself. Should it be called on both?

Reply:
2c318ca1b42b0b806ba5a4e328d193a0fba9581f makes it execute on the package itself as well.

Reply:
It’s already called on the package itself because args to `buildRustPackage` are passed through to `stdenv`.